src/tests/ftest/io/io_consistency.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2020 Intel Corporation.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+  The Government's rights to use, modify, reproduce, release, perform, display,
+  or disclose this software are subject to the terms of the Apache License as
+  provided in Contract No. B609815.
+  Any reproduction of computer software, computer software documentation, or
+  portions thereof marked with this legend must also reproduce the markings.
+"""
+
+from ior_test_base import IorTestBase
+
+
+class IoConsistency(IorTestBase):
+    """Test class Description: Verify data consistency using different
+                               middlewares. In this case, using DFS and POSIX.
+
+    :avocado: recursive
+    """
+
+    def tearDown(self):
+        """Tear down each test case."""
+        try:
+            if self.dfuse:
+                self.dfuse.stop()
+        finally:
+            # Stop the servers and agents
+            super(IoConsistency, self).tearDown()
+
+    def test_ioconsistency(self):
+        """Jira ID: DAOS-4778.
+        Test Description:
+            Run IOR first using DFS and then using POSIX to verify data
+            consistency.
+        Use Cases:
+            Create a pool
+            Create POSIX type container.
```

Comment:
Maybe I'm just not knowledgeable enough.
Do you create the pool and the container through run_ior_with_pool? If so, what if api_flag[0] == "POSIX"? Can you run ior without a pool?
I don't know if this is the right PR to talk bout this, but I'm not sure if it's a good idea to combine pool create and run ior in the same method, especially with the update flag. Does the method work with update=False when the pool isn't created? It gets harder to understand and maintain if a method does two unrelated tasks.

Reply:
Okay. Thanks for the explanation.